pupil_src/shared_modules/gaze_mapping/optimization_calibration/bundle_adjustment.py

```diff
@@ -13,8 +13,9 @@
 import numpy as np
 from scipy import optimize as scipy_optimize, sparse as scipy_sparse
 
-from calibration_routines.optimization_calibration import utils
+from . import utils
 
+# TODO: What is the difference between this class and head_pose_tracker/function/bundle_adjustment.py?
```

Comment:
@romanroibu I talked to @ChingT. There are significant differences that cannot be merged easily. Let's keep both files as they are.